fastly/ftp.go

```diff
@@ -166,19 +168,20 @@ type UpdateFTPInput struct {
 	// Name is the name of the FTP to update.
 	Name string
 
-	NewName           string `form:"name,omitempty"`
-	Address           string `form:"address,omitempty"`
-	Port              uint   `form:"port,omitempty"`
-	Username          string `form:"user,omitempty"`
-	Password          string `form:"password,omitempty"`
-	Path              string `form:"path,omitempty"`
-	Period            uint   `form:"period,omitempty"`
-	FormatVersion     uint   `form:"format_version,omitempty"`
-	GzipLevel         uint8  `form:"gzip_level,omitempty"`
-	Format            string `form:"format,omitempty"`
-	ResponseCondition string `form:"response_condition,omitempty"`
-	TimestampFormat   string `form:"timestamp_format,omitempty"`
-	Placement         string `form:"placement,omitempty"`
+	NewName           string  `form:"name,omitempty"`
+	Address           string  `form:"address,omitempty"`
+	Port              uint    `form:"port,omitempty"`
+	PublicKey         *string `form:"public_key,omitempty"`
```

Comment:
I'd prefer it if the structs interface was consistent, so don't think we should introduce the pointer pattern, even for a new field.